### ISSUE TITLE
build fails on windows 

### ISSUE THREAD

**Issue by killerpommes** [Dykam/tabbychat#56]
Hey i tried to build tabbychat on windows with "gradlew setupCIWorkspace build" but i get this error. I have no idea what i did wrong.

```
Starting process 'command 'git''. Working directory: C:\tabbychat-master Command: git describe --tags --abbrev=0Starting process 'command 'git''. Working directory: C:\tabbychat-master Command: git describe --tags --abbrev=0

An attempt to initialize for well behaving parent process finished.An attempt to initialize for well behaving parent process finished.

****************************
 Powered By MCP:            
 http://mcp.ocean-labs.de/  
 Searge, ProfMobius, Fesh0r,
 R4wk, ZeuX, IngisKahn      
Resolving:  MCP Data version : {MCP_VERSION}
Failed to notify ProjectEvaluationListener.afterEvaluate(), but primary configuration failure takes precedence.
org.gradle.api.ProjectConfigurationException: You must set the Minecraft Version!
    at net.minecraftforge.gradle.user.patch.UserPatchExtension.getApiVersion(UserPatchExtension.java:64)
    at net.minecraftforge.gradle.user.patch.UserPatchExtension_Decorated.getApiVersion(Unknown Source)
    at net.minecraftforge.gradle.user.patch.UserPatchBasePlugin.getApiVersion(UserPatchBasePlugin.java:241)
    at net.minecraftforge.gradle.user.patch.UserPatchBasePlugin.getApiVersion(UserPatchBasePlugin.java:31)
    at net.minecraftforge.gradle.user.UserBasePlugin.resolve(UserBasePlugin.java:269)
    at net.minecraftforge.gradle.user.UserBasePlugin.resolve(UserBasePlugin.java:108)
    at net.minecraftforge.gradle.delayed.DelayedBase.resolve(DelayedBase.java:73)
    at net.minecraftforge.gradle.delayed.DelayedString.call(DelayedString.java:24)
    at net.minecraftforge.gradle.common.BasePlugin.afterEvaluate(BasePlugin.java:151)
    at net.minecraftforge.gradle.user.UserBasePlugin.afterEvaluate(UserBasePlugin.java:885)
    at net.minecraftforge.gradle.common.BasePlugin$2.execute(BasePlugin.java:102)
    at net.minecraftforge.gradle.common.BasePlugin$2.execute(BasePlugin.java:98)
    at org.gradle.listener.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:105)
    at org.gradle.listener.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:94)
    at org.gradle.listener.BroadcastDispatch.dispatch(BroadcastDispatch.java:79)
    at org.gradle.listener.BroadcastDispatch.dispatch(BroadcastDispatch.java:31)
    at org.gradle.messaging.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
    at com.sun.proxy.$Proxy13.afterEvaluate(Unknown Source)
    at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:67)
    at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:61)
    at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:507)
    at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:82)
    at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuildConfigurer.java:31)
    at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:142)
    at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:113)
    at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:81)
    at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildController.run(InProcessBuildActionExecuter.java:64)
    at org.gradle.launcher.cli.ExecuteBuildAction.run(ExecuteBuildAction.java:33)
    at org.gradle.launcher.cli.ExecuteBuildAction.run(ExecuteBuildAction.java:24)
    at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:35)
    at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:26)
    at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:50)
    at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:171)
    at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:201)
    at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:174)
    at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:170)
    at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:139)
    at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
    at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
    at org.gradle.launcher.Main.doAction(Main.java:46)
    at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
    at org.gradle.launcher.Main.main(Main.java:37)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:50)
    at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:32)
    at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:30)
    at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:127)
    at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:58)
Caused by: java.lang.NullPointerException
    ... 56 more

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred evaluating root project 'tabbychat-master'.
> A problem occurred starting process 'command 'git''

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output.

BUILD FAILED

Total time: 5.82 secs
Stopped 0 compiler daemon(s).
```

**Comment by mattmess1221** [comment]
Try installing git.

**Comment by insert3coins** [comment]
also do this:

gradlew setupCIWorkspace
gradlew build

works for me

**Comment by mattmess1221** [comment]
@tehface that's not the issue. It's failing because it can't run git, which is used to get the revision. I may do a PR to FG to return null if git isn't on the path.

**Comment by insert3coins** [comment]
ya i see that now.. my bad lols

**Comment by killerpommes** [comment]
what version of git should i use i installed git an put it into the PATH. 
Now i get this Error
FAILURE: Build failed with an exception.
- Where:
  Build file 'C:\tabbychat-master\build.gradle' line: 72
- What went wrong:
  A problem occurred evaluating root project 'tabbychat-master'.
  
  > Process 'command 'git'' finished with non-zero exit value 128

**Comment by mattmess1221** [comment]
Let me rephrase that. Install git then clone the repo. Build using that.

**Comment by killerpommes** [comment]
i got i working.
I cloned the repo instead of downloading the zip file from github
Thanks for the help

**Comment by mattmess1221** [comment]
MinecraftForge/ForgeGradle#107 would prevent this.
